public/app/plugins/panel/geomap/types.ts

```diff
@@ -43,3 +44,23 @@ export interface GeomapPanelOptions {
   basemap: MapLayerOptions;
   layers: MapLayerOptions[];
 }
+export interface FeatureStyleConfig {
+  shape?: string;
+  fill?: Fill; //eventually be ColorDimensionConfig
+  stroke?: Stroke; //eventually be ColorDimensionConfig
+  strokeWidth?: number;
+  rule?: FeatureRuleConfig;
+}
+export interface FeatureRuleConfig {
+  property: string;
+  operations: ComparisonOperations;
```

Comment:
```suggestion
  operation: ComparisonOperation;
```
Can we use singular?